Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -22,12 +22,12 @@ public ContactPolicies(IContactPoliciesService contactPoliciesService)
         [HttpGet("/accounts/{accountName}/settings")]
         public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         {
-            // TODO: replace this validation to get UserId by accountName
-            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
-
-            if (contactPoliciesSettings == null)
+            var userFounded = await _contactPoliciesService.GetIdUserByAccountName(accountName);
+            if (userFounded == null)
```

Comment:
In my opinion, using the a real meaning in the variable name is better:

```suggestion
            var idUser = await _contactPoliciesService.GetIdUserByAccountName(accountName);
            if (idUser == null)
```

Reply:
Resolved!